cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,63 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import SettingPageObject from '../support/pages/settings.pageObject';
+import updatedData from '../support/updatedData';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+
+const settings = new SettingPageObject();
+const newData = updatedData;
+const profilePage = new ProfilePageObject();
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+
 describe('Settings page', () => {
+  let user;
   before(() => {
-
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
   });
 
   beforeEach(() => {
-
+    cy.task('db:clear');
+    cy.login(user.email, user.username, user.password);
+    settings.visit();
   });
 
   it('should provide an ability to update username', () => {
-
+    settings.UsernameInput.clear();
+    settings.UsernameInput.type(newData.username);
```

Comment:
```suggestion
    settings.UsernameInput.clear();
    settings.UsernameInput.type(newData.username);
```
this are not actual page object methods. You need to implement these 2 functions inside page object method and it should look like this settings.fillTheUsernameField(newData.username);
Check all the same line of code